app/views/layouts/application.html.erb

```diff
@@ -8,6 +8,7 @@
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-9ndCyUaIbzAi2FUVXJi0CjmCapSmO7SnpJef0486qhLnuZ2cdeRhO02iuK6FUUVM" crossorigin="anonymous">
```

Comment:
Don't use CDN. Please try these steps bellow to setup bootstrap (https://blog.eq8.eu/til/how-to-use-bootstrap-5-in-rails-7.html):
- Add ```gem 'bootstrap', '~> 5.1.3'```
- Bundle
- Run ```bin/importmap pin bootstrap```
- Add ```@import "bootstrap";``` to ```app/assets/stylesheets/application.scss ```